exercises/parallel-letter-frequency/benches/benchmark.rs

```diff
@@ -0,0 +1,73 @@
+#![feature(test)]
+extern crate parallel_letter_frequency;
+extern crate test;
+
+use std::collections::HashMap;
+use test::Bencher;
+
+#[bench]
+fn bench_sequential(b: &mut Bencher) {
+    let texts = all_texts();
+    b.iter(|| frequency(&texts));
+}
+
+#[bench]
+fn bench_parallel(b: &mut Bencher) {
+    let texts = all_texts();
+    b.iter(|| parallel_letter_frequency::frequency(&texts, 3));
+}
+
+/// Simple sequential char frequency. Can it be beat?
+pub fn frequency(texts: &[&str]) -> HashMap<char, usize> {
```

Comment:
I can agree with that. I can't speak for every single track, but the Go track also provides students with a frequency function: https://github.com/exercism/xgo/blob/master/exercises/parallel-letter-frequency/frequency.go

Edit: Oh... you [already mentioned](https://github.com/exercism/xrust/pull/245#issuecomment-272192375) that you took this pattern from the Go track. I should really read all of the comments on this PR before jumping in to review...